src/redux/admins/reducer.js

```diff
@@ -53,6 +59,44 @@ const reducer = (state = INITIAL_STATE, action) => {
         error: action.payload,
         list: []
       };
+    case POST_ADMINS_PENDING:
+      return {
+        ...state,
+        isLoading: true
+      };
+    case POST_ADMINS_FULLFILLED:
+      return {
+        ...state,
+        list: [...state.list, action.payload],
+        error: '',
+        isLoading: false
+      };
+    case POST_ADMINS_ERROR:
+      return {
+        ...state,
+        isLoading: false,
+        error: action.payload,
+        list: []
+      };
+    case PUT_ADMINS_PENDING:
+      return {
+        ...state,
+        isLoading: true
+      };
+    case PUT_ADMINS_FULLFILLED:
+      return {
+        ...state,
+        list: [...state.list, action.payload],
```

Comment:
We should change this logic because we are adding a new admin to the list and we should update this.
we should have something like:
```suggestion
        list: state.list.map((admin) => {
          if (admin._id === action.payload._id) {
            return action.payload;
          }
          return admin;
        }),
```

Reply:
Logic changed, .map implemented